hack/run-e2e-conformance-virtual-cluster.sh

```diff
@@ -120,6 +120,9 @@ controller_ip=`kubectl get node -o wide | grep ctlp | awk '{print $6}'`
 insecure_registry="[[registry]]
 location = \"$controller_ip:5000\"
 insecure = true
+
+[aliases]
+\"golang\" = \"docker.io/library/golang\"
```

Comment:
why do we need this one ?
(commit msg doesnt mention this change)

Reply:
let me add it to the commit this is to avoid the CI on new nodes to stuck asking what is the full image name.
we can also change the this in our docker images to avoid this issue